server/src/main/java/org/opensearch/common/cache/stats/CacheStatsHolder.java

```diff
@@ -41,9 +44,12 @@ public class CacheStatsHolder {
     // To avoid sync problems, obtain a lock before creating or removing nodes in the stats tree.
     // No lock is needed to edit stats on existing nodes.
     private final Lock lock = new ReentrantLock();
+    // The name of the cache type using these stats
+    private final String storeName;
```

Comment:
It's passed in from ICache implementations. For example if we're using on-heap cache it's "opensearch_onheap", if we're using the tiered cache it's "tiered_spillover", etc. Sagar had thought this would be good to include in the API response